### PR TITLE
Add Python logging sample

### PR DESCRIPTION
**Pull request by jhuleatt**
Translated from the [Node sample](https://github.com/firebase/functions-samples/blob/main/Node/quickstarts/monitor-cloud-logging/functions/index.js)

### CODE REVIEW

**Review by kevinthecheung — APPROVED** [review]

File: Python/quickstarts/monitor-cloud-logging/functions/main.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Welcome to Cloud Functions for Firebase for Python!
+# To get started, simply uncomment the below code or create your own.
```
Comment (egilmorez) [inline Python/quickstarts/monitor-cloud-logging/functions/main.py:2]: Please don't use "simply" (nor for that matter "please" :D )

**Review by egilmorez — APPROVED** [review]
LGTM with a nit, thanks!

File: Python/quickstarts/monitor-cloud-logging/functions/main.py (hunk 1 above)
Comment (jhuleatt) [reply Python/quickstarts/monitor-cloud-logging/functions/main.py:2]: oops, I'll take that whole comment out. it comes from the CLI template: https://github.com/firebase/firebase-tools/blob/e53c50dff3542a8932cbc668ea326defb0118069/templates/init/functions/python/main.py#L2

File: Python/quickstarts/monitor-cloud-logging/functions/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+firebase_functions~=0.1.0
```
[the remark this answers is not in the document]
Comment (jhuleatt) [reply Python/quickstarts/monitor-cloud-logging/functions/requirements.txt:1]: good catch, looks like the CLI's template is out of date: https://github.com/firebase/firebase-tools/blob/master/templates/init/functions/python/requirements.txt

[the remark this answers is not in the document]
Comment (jhuleatt) [reply Python/quickstarts/monitor-cloud-logging/functions/requirements.txt]: updated to 0.1.2

File: Python/quickstarts/monitor-cloud-logging/functions/main.py (hunk 1 above)
Comment (jhuleatt) [reply Python/quickstarts/monitor-cloud-logging/functions/main.py]: removed

**Pull request merged**